lib/web_ui/test/engine/image/image_test.dart

```diff
@@ -0,0 +1,31 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:test/bootstrap/browser.dart';
+// import 'package:test/test.dart';
+// import 'package:ui/ui.dart' as ui;
+
+void main() {
+  internalBootstrapBrowserTest(() => testMain);
+}
+
+Future<void> testMain() async {
+  // TODO(polina-c): uncomment the test when bug is fixed:
+  // https://github.com/flutter/engine/pull/35791
+  // test('toImage succeeds', () async {
+  //   final ui.Image image = await _createImage();
+  //   expect(image.runtimeType.toString(), equals('HtmlImage'));
+  //   image.dispose();
+  // });
```

Comment:
Maybe keep the code uncommented, but `skip` it? This way at least the code can be analyzed.

Reply:
done